spec/api/v1/dropdown_data_spec.rb

```diff
@@ -63,13 +65,15 @@
         EXPENSE_REASONS_ENDPOINT => API::Entities::ExpenseReasonSet.represent(ExpenseType.reason_sets).to_json,
         DISBURSEMENT_TYPE_ENDPOINT => API::Entities::DisbursementType.represent(DisbursementType.active).to_json,
         TRANSFER_STAGES_ENDPOINT => API::Entities::SimpleKeyValueList.represent(Claim::TransferBrain::TRANSFER_STAGES.to_a).to_json,
-        TRANSFER_CASE_CONCLUSIONS_ENDPOINT => API::Entities::SimpleKeyValueList.represent(Claim::TransferBrain::CASE_CONCLUSIONS.to_a).to_json
+        TRANSFER_CASE_CONCLUSIONS_ENDPOINT => API::Entities::SimpleKeyValueList.represent(Claim::TransferBrain::CASE_CONCLUSIONS.to_a).to_json,
+        CASE_STAGE_ENDPOINT => API::Entities::CaseStage.represent(CaseStage.all).to_json,
       }
     end
 
     before do
       seed_fee_schemes
       seed_case_types
+      seed_case_stage
```

Comment:
my error, but this should be `seed_case_stages` (plural). you could add a commit to fix my typo

Reply:
👍 have amended to seed_case_stages